src/main/java/zd/zero/waifu/motivator/plugin/assets/VisualMotivationAssetProvider.kt

```diff
@@ -53,13 +54,24 @@ object VisualMotivationAssetProvider {
         audibleAssetDefinition: AudibleMotivationAssetDefinition
     ): MotivationAsset =
         MotivationAsset(
-            textualAssetDefinition.title,
+            "&nbsp;&nbsp;&nbsp;${textualAssetDefinition.title}",
             """
-                 <img src='${visualAssetDefinition.imagePath}' alt='${visualAssetDefinition.imageAlt}' width='256' />
-                 <br><br><br><br><br><br><br><br><br><br>
-                 <p>${textualAssetDefinition.message}</p>
-                """.trimIndent(),
+                <div style="margin: 5px 5px 5px 10px;${getExtraStyles(visualAssetDefinition.imageDimensions)}" >
+                    <img src='${visualAssetDefinition.imagePath}' alt='${visualAssetDefinition.imageAlt}'/>
+                </div>
+            """.trimIndent(),
             audibleAssetDefinition.soundFile,
             arrayOf()
         )
+
+    private fun getExtraStyles(imageDimensions: ImageDimension): String =
+        if (SystemInfo.isLinux) {
```

Comment:
```suggestion
        if (!SystemInfo.isMac) {
```

![Macos is special](https://i.imgflip.com/4alfnd.jpg)